core/transport/src/grpc/job_master.proto

```diff
@@ -55,7 +56,7 @@ message JobCommand {
 
 message RunTaskCommand {
   optional int64 jobId = 1;
-  optional int32 taskId = 2;
+  optional int64 taskId = 2;
```

Comment:
Is this change backward compatible?

Reply:
This one doesn't actually matter because this is a Master to Worker communication.